src/components/NewsRoomList.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import FilterList from "./FilterList";
+import NewsRoomCard from "./NewsRoomCard";
+import { newsItems } from "../files/NewsItems";
+import ListCounter from "./ListCounter";
+import useNewsRoom from "../hooks/useNewsRoom";
+import { Alert, Button } from "@baltimorecounty/dotgov-components";
+
+const NewsRoomList = () => {
+  const { hasError, newsRoomItems = [], isLoading } = useNewsRoom();
```

Comment:
this is not blocking, but I think this hook makes more sensse as useNews. The reason being this is just fluffy name we gave it, but really its just news. And we will also use this likely for other types of news, like police and fire news that will be in upcoming work.

Reply:
changed